script/Ciel/Day 5/05_01_CIEL05_1B.txt

```diff
@@ -64,12 +64,12 @@ The man named Haru pushes the button for the elevator.
 -- Page 65, Offset 11030.
 -- 「……っ」
 // This has 「」 but... I'm straining my ears here and I don't here any voices, just elevator SFX
-%{i}Rattle%{/i}.
+Through the darkness―――
```

Comment:
Kinda prefer original of these two lines. Avoids having to splice the sentence.

Reply:
Really don't like the sfx mixing, it just reads kinda awkwardly. The ones that are coming from jp onomatopoeia i understand, but these i'd rather avoid.
If we want to just stick with one word, it should just be "..."

Reply:
Fine with just ellipsis here and reverting next line.